debian/rules

```diff
@@ -208,6 +208,9 @@ install: build
 	##
 	mv debian/temp/bin/* debian/opennms-common/opt/opennms/bin/
 	find debian/opennms-common/opt/opennms/bin -type f -execdir chmod 755 {} \;
+	find debian/temp/etc/ -type f -print | \
+                grep -v -e '.jasper' -e examples -e profile.cfg -e org.ops4j.pax.url.mvn.cfg -e custom.properties -e config.properties -e opennms.properties -e org.apache.karaf -e log4j2.xml -e startup.properties -e THIRD-PARTY.txt -e overrides.properties | \
```

Comment:
The .jasper files are the compiled versions of the .jrxml report templates. We either include both or exclude both.

I'm not sure if we have people who have customized these. It might be the safest bet to let the user decide if he wants to use the ones coming with the package or wants to keep the one he has modified.

Reply:
Looking at the code, it looks like OpenNMS generates .jasper at runtime from .jrxml file. Code [here](https://github.com/OpenNMS/opennms/blob/1733cbe624378ccd34bb96450e326d26cec9c1e8/features/reporting/jasper-reports/src/main/java/org/opennms/reporting/jasperreports/svclayer/JasperReportService.java#L591)

So this looks good to me.